python/ray/tune/tests/test_checkpoint_manager.py

```diff
@@ -94,6 +94,7 @@ def testBestCheckpoints(self):
             _TuneCheckpoint(_TuneCheckpoint.PERSISTENT, i, self.mock_result(i, i))
             for i in range(16)
         ]
+        random.seed(1)
```

Comment:
Do we need this? If we're failing for certain cases, doesn't this indicate a bug?

Reply:
We shouldn't have stochastic tests. We should either set the seed or test all the combinations